### ISSUE TITLE
NRatings is not pulling race stats for the Camping World Truck Series races since the Bristol Dirt race.

### ISSUE THREAD

**Issue by JRFranklin62** [64Soft/NRatings.Client#36]
The issue is in the subject line.  Also, it doesn't appear to be pulling race info for the ARCA Menards East and West Series either.

Thanks in advance for looking into this.

J.R.

**Comment by KBMfan** [comment]
Bumping this post as I can also confirm multiple series are missing 2022 race results. The ARCA Menards Series hasn't been updated in over 3 months, for example. 

Hopefully the creator adds a feature where you can enter real-life results on your own so the users aren't waiting on one person, whoever that may be, to have the time to put the info into the program. Not sure if that's even possible, but it'd nullify some continuity issues.

**Comment by 64Soft** [comment]
Closing, as this is a duplicate of #35
